webservices/tasks/download.py

```diff
@@ -38,7 +38,7 @@ def call_resource(path, qs):
     return {
         "path": path,
         "qs": qs,
-        "name": get_s3_name(path, qs),
+        "name": get_s3_name(path, qs.encode('utf-8')),
```

Comment:
I encode the query string back to bytes to ensure consistency with previously cached names